.neuro/hypertrain.yml

```diff
@@ -0,0 +1,64 @@
+kind: batch
+
+params:
+  token:
+    descr: W&B auth token
+
+volumes:
+  data:
+    remote: storage:$[[ flow.project_id ]]/data
+    mount: /project/data
+  code:
+    remote: storage:$[[ flow.project_id ]]/src
+    mount: /project/code
+  config:
+    remote: storage:$[[ flow.project_id ]]/config
+    mount: /project/config
+    read_only: True
+  notebooks:
+    remote: storage:$[[ flow.project_id ]]/notebooks
+    mount: /project/notebooks
+  results:
+    remote: storage:$[[ flow.project_id ]]/results
+    mount: /project/results
+  project:
+    remote: storage:$[[ flow.project_id ]]
+    mount: /project
+
+images:
+  myimage:
+    ref: image:$[[ flow.project_id ]]
+    dockerfile: $[[ flow.workspace ]]/Dockerfile
```

Comment:
I would suggest removing all except "ref", not to confuse users.
We should also state somewhere at the end of the readme, that `neuro-flow build` refers only to the "live" neuro-flow mode of operation.

Reply:
Done